### PR TITLE
23.05 Release Blog

### PR DESCRIPTION
**Pull request by aFinchy**
## Change summary

Created 23.05 Release Blog as per LF Request. 

Images that was handed are a bit low res so may appear not too good of quality. Merging to main as Dev has issues with blogs usually. 

### Submission Checklist:

* [ ] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [ ] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [ ] **Consistency** - Does the content consistently follow the [Style Guide](https://o3de.org/docs/contributing/to-docs/style-guide/quick-reference)?
* [ ] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

File: content/blog/posts/23-05-release.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+**Increased Usability and Performance for a Diversity of 3D Applications**
+
+This latest release makes it even easier for developers to harness O3DE for a multitude of 3D projects:
+
+●	**Craft your own unique shaders and materials** using [Material Canvas](https://github.com/o3de/sig-graphics-audio/issues/51), a fresh, easy-to-use visual scripting interface that blends the power of Script Canvas and the Material Editor. 
+●	**Customize the render pipeline more easily than ever** with a Material Pipeline.
+●	**Manage assets like a boss** using the Asset Browser, with different layout options, file operations, an asset inspector panel, and other operations.
+●	**Install multiple versions of O3DE on the same drive**, with projects aware of which O3DE version they were created in and last used with.
+●	**Specify compatible platforms for gems you create**, which can be filtered in the Gem Catalog.
+●	**Add menus, hotkeys and contextual actions from Gems and Python scripts** with enhanced extensibility of the Editor though a new [Action Manager API](https://www.o3de.org/docs/user-guide/action-manager/).
```
Comment (micronAMZN) [inline content/blog/posts/23-05-release.md:39]: Fixing this list.

```suggestion
*	**Craft your own unique shaders and materials** using [Material Canvas](https://github.com/o3de/sig-graphics-audio/issues/51), a fresh, easy-to-use visual scripting interface that blends the power of Script Canvas and the Material Editor. 
*	**Customize the render pipeline more easily than ever** with a Material Pipeline.
*	**Manage assets like a boss** using the Asset Browser, with different layout options, file operations, an asset inspector panel, and other operations.
*	**Install multiple versions of O3DE on the same drive**, with projects aware of which O3DE version they were created in and last used with.
*	**Specify compatible platforms for gems you create**, which can be filtered in the Gem Catalog.
*	**Add menus, hotkeys and contextual actions from Gems and Python scripts** with enhanced extensibility of the Editor though a new [Action Manager API](https://www.o3de.org/docs/user-guide/action-manager/).
```

File: content/blog/posts/23-05-release.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+●	**Animations:** Along with a more consistent [Animation Editor](https://github.com/o3de/o3de/issues/10666) experience and a more robust, streamlined [Animation Asset Import](https://github.com/o3de/o3de/issues/12387) process, the Animation Editor AnimGraph now has a [Performance Visualizer](https://github.com/o3de/o3de/pull/13490) to help users profile and optimize their AnimGraphs.
+●	**Terrain System:** A new [paintbrush](https://www.o3de.org/docs/user-guide/components/reference/paintbrush/paintbrush/) tool enables users to paint inside the viewport to create or modify terrain, while a new [Terrain Developer Guide](https://www.o3de.org/docs/user-guide/visualization/environments/terrain/terrain-developer-guide/) aids developers in using and extending the terrain system.
+●	**VR/XR and Mobile Support:** New [OpenXR and OpenXRVk Gems](https://github.com/o3de/o3de/issues/12372) deliver support for OpenXR-compatible devices, such as Meta Quest 2, including stereoscopic rendering, while the addition of half-float support for the Atom renderer improves the performance on mobile devices.
```
Comment (micronAMZN) [inline content/blog/posts/23-05-release.md:66]: Fixing this list

```suggestion
*	**Physics:** In addition to improvements in multiple PhysX authoring workflows, users can now enable support for [PhysX 5.1](https://github.com/o3de/o3de/issues/13624) with a simple flag in O3DE, which shows a 15% increase in simulation performance compared with PhysX 4, as well as improvements in multiple PhysX authoring workflows.
*	**Animations:** Along with a more consistent [Animation Editor](https://github.com/o3de/o3de/issues/10666) experience and a more robust, streamlined [Animation Asset Import](https://github.com/o3de/o3de/issues/12387) process, the Animation Editor AnimGraph now has a [Performance Visualizer](https://github.com/o3de/o3de/pull/13490) to help users profile and optimize their AnimGraphs.
*	**Terrain System:** A new [paintbrush](https://www.o3de.org/docs/user-guide/components/reference/paintbrush/paintbrush/) tool enables users to paint inside the viewport to create or modify terrain, while a new [Terrain Developer Guide](https://www.o3de.org/docs/user-guide/visualization/environments/terrain/terrain-developer-guide/) aids developers in using and extending the terrain system.
*	**VR/XR and Mobile Support:** New [OpenXR and OpenXRVk Gems](https://github.com/o3de/o3de/issues/12372) deliver support for OpenXR-compatible devices, such as Meta Quest 2, including stereoscopic rendering, while the addition of half-float support for the Atom renderer improves the performance on mobile devices.
```

File: content/blog/posts/23-05-release.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+author: Royal O’Brien
+blog_img: "/images/blog/announcement_thumbnail.jpg"
+full_img: ""
+---
+
+We’re excited to announce the newest release of the [Open 3D Engine (O3DE)](https://www.o3de.org/), Version 23.05, which is jam-packed with enhancements and new features. In addition to usability and stability improvements, new capabilities will help enable developers to build games, robotic simulations and other 3D applications alike.
+
+To celebrate, we’re holding our first-ever [O3DE Jam](https://o3d.foundation/blog/calling-all-open-source-3d-developers-lets-jam-in-the-wasteland/) beginning tomorrow where the community can come together to build some seriously cool 3D experiences using all the new features of this latest release.
+
+Read on for a few of the highlights, get the details in the [release notes](https://www.o3de.org/docs/release-notes/22-10-0/), and join the jam!
```
Comment (micronAMZN) [inline content/blog/posts/23-05-release.md:14]: fixing release notes link

```suggestion
Read on for a few of the highlights, get the details in the [release notes](https://www.o3de.org/docs/release-notes/2305-0-release-notes), and join the jam!
```

**Review by micronAMZN — COMMENTED** [review]
Merging these suggestions myself and texting the deploy preview.

**Review by micronAMZN — APPROVED** [review]

**Review by chanmosq — APPROVED** [review]

**Pull request merged**